addons/point_of_sale/static/tests/tours/PosComboTour.js

```diff
@@ -42,6 +42,17 @@ registry.category("web_tour.tours").add("PosComboPriceTaxIncludedTour", {
         ...ProductScreen.clickOrderline("Combo Product 8"),
         ...ProductScreen.selectedOrderlineHas("Combo Product 8", "1.0", "30.00"),
 
+        // check that there is no price shown on the parent line
+        ...inLeftSide({
+            trigger: `.order-container`,
```

Comment:
You should be able to use the `negate` function to avoid using the run.
https://github.com/odoo/odoo/blob/f7e3ada5ed290a6f418e4e1a2abddfb9f242270c/addons/point_of_sale/static/tests/tours/helpers/utils.js#L21-L23
Besides that it LGTM